### PR TITLE
EE-1204: Update to Project Notification Document Card

### PR DESCRIPTION
**Pull request by danieltruong**

### CODE REVIEW

File: src/app/project-notifications/project-notification-documents-table-rows/project-notification-documents-table-rows.component.ts [hunk 1]
```diff
@@ -0,0 +1,56 @@
...
+    }
+
+    idToList(id: string) {
+        if (!id) {
+            return '-';
+        }
+        // Grab the item from the constant lists, returning the name field of the object.
+        let item = this.lists.filter(listItem => listItem._id === id);
+        if (item.length !== 0) {
+            return item[0].name;
```
Comment (marklise) [inline src/app/project-notifications/project-notification-documents-table-rows/project-notification-documents-table-rows.component.ts:36]: Variable should be `items` not `item`

File: src/app/project-notifications/project-notification-documents-table-rows/project-notification-documents-table-rows.component.ts [hunk 2]
```diff
@@ -0,0 +1,56 @@
...
+        } else {
+            return '-';
+        }
+    }
+
+    goToItem(item) {
+        let filename = item.documentFileName || item.displayName || item.internalOriginalName;
```
Comment (marklise) [inline src/app/project-notifications/project-notification-documents-table-rows/project-notification-documents-table-rows.component.ts:43]: `const` not `let`

File: src/app/project-notifications/project-notifications-table-rows/project-notifications-table-rows.component.html [hunk 3]
```diff
@@ -1,89 +1,100 @@
...
+            <h4>{{(rowData.name.toUpperCase() || '-')}}</h4>
+          </div>
+          <div class="col-sm-12 col-md-6">
+            <a *ngIf="rowData.commentPeriod?.daysRemainingCount > 0"
+              class="btn btn-primary view-pcp-btn"
+              title="View this Public Comment Period"
+              [routerLink]="['/pn', rowData._id, 'cp', rowData.commentPeriod?._id]">
+              <div class="pcp-indicator pcp-indicator-active"></div>
+              Open Comment Period - View/Submit
+            </a>
```
Comment (marklise) [inline src/app/project-notifications/project-notifications-table-rows/project-notifications-table-rows.component.html:18]: Good candidate in the future for a component, not to be addressed here.

**Review by marklise — CHANGES_REQUESTED** [review]
Mobile view does not look usable

File: src/app/project-notifications/project-notification-documents-table-rows/project-notification-documents-table-rows.component.ts (hunk 1 above)
Comment (danieltruong) [reply src/app/project-notifications/project-notification-documents-table-rows/project-notification-documents-table-rows.component.ts]: 1ccaec1

File: src/app/project-notifications/project-notifications-table-rows/project-notifications-table-rows.component.html (hunk 3 above)
Comment (danieltruong) [reply src/app/project-notifications/project-notifications-table-rows/project-notifications-table-rows.component.html:19]: 1ccaec1

File: src/app/project-notifications/project-notification-documents-table-rows/project-notification-documents-table-rows.component.ts (hunk 2 above)
Comment (danieltruong) [reply src/app/project-notifications/project-notification-documents-table-rows/project-notification-documents-table-rows.component.ts]: 1ccaec1

**Review by MaxWardle — APPROVED** [review]
Looks good 👍

**Review by BcGovNeal — APPROVED** [review]

**Review by marklise — APPROVED** [review]

**Pull request merged**